index.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html lang="en">
```

Comment:
- [ ] Kindly add a readme file to your branch and make sure you customized every section of the branch for your project. Please use the readme template provided by microverse.
- [ ] Please in all future projects, make sure your commit messages are small enough to describe them in one sentence. The commit message should be written in the imperative tense and capitalized. For example `Add markup to index.html`

Reply:
@Kamba56 changes effected

Reply:
Hello @Techcolony, just like the previous reviewer suggested, please make sure that all the required sections of your readme are present in your readme. Please fix this 🙏 as it would help to make your readme look more professional.